[SRVKS-1182] Add the new kourier bootstrap documentation (#79689)
openshift/openshift-docs: modules/enabling-administrator-interface-access.adoc

> :_content-type: PROCEDURE

Reviewer: same comment above about metadata

[id="enabling-administrator-interface-access_{context}"]
= Enabling administrator interface access

You can change the envoy bootstrap configuration to enable access to the administrator interface.

[IMPORTANT]
====
This procedure assumes sufficient knowledge of Knative, as changing envoy bootstrap configuration might result in Knative failure. Red Hat does not support custom configurations that are not tested or shipped with the product.
====

.Prerequisites

* You have installed the {ServerlessOperatorName} and Knative Serving.
* You have cluster administrator permissions on {ocp-product-title}, or you have cluster or dedicated administrator permissions on {rosa-product-title} or {dedicated-product-title}.

.Procedure

> * To enable administrator interface access, locate this configuration in your bootstrapping configuration map:

Reviewer: ```suggestion
. To enable administrator interface access, locate this configuration in your bootstrapping configuration map:
```

Reviewer: Since you have a second step below when you say "Apply the configuration in two places", I think this should be a 1. step instead of an unordered item.

+
[source,terminal]
----
pipe:
  path: /tmp/envoy.admin
----
+
Substitute it with the following configuration:
+
[source,terminal]
----
socket_address:
  address: 127.0.0.1
  port_value: 9901
----
+

> This configuration enables access to the Envoy admin interface on the loopback address (127.0.0.1) and port 9901.

Reviewer: Could this be a code callout for `socket_address` instead of a sentence after the code? Especially since you have another sentence on line 38.

+

> This change needs to be applied in two places. The first is in the `service_stats` cluster configuration:

Reviewer: ```suggestion
. Apply the `socket_address` configuration in the `service_stats` cluster configuration and in the `admin` configuration:
.. The first is in the `service_stats` cluster configuration:
```

Reviewer: I suggest making this a numbered 2. step and then sub-stepping the two applications.

+
[source,terminal]
----
clusters:
  - name: service_stats
    connect_timeout: 0.250s
    type: static
    load_assignment:
      cluster_name: service_stats
      endpoints:
        lb_endpoints:
          endpoint:
            address:
              socket_address:
                address: 127.0.0.1
                port_value: 9901
----
+

> The second is in the `admin` configuration:

Reviewer: ```suggestion
.. The second is in the `admin` configuration:
```
